src/events/sensor/events/created.event.ts

```diff
@@ -0,0 +1,29 @@
+import { EventType } from "./event-type";
+import { Event } from "../../../event-store/event";
+import { LocationBody } from '../../../commands/sensor/models/bodies/location-body';
+
+
+export class SensorCreated extends Event {
+  constructor(aggregatedId: string, nodeId: string, ownerIds: Array<string>, 
+    name: string, location: LocationBody, aim: string, description: string, 
+    manufacturer: string, active: boolean, observationArea: object, 
+    documentationUrl: string, category: string, theme: string, typeName: string, 
+    typeDetails: object) {
+    super(`sensor-${aggregatedId}`, EventType.SensorCreated, {
+      nodeId,
```

Comment:
`sensorId` ook toevoegen als veld, anders is dat alleen in de stream opgeslagen. Het is wellicht dubbel, maar als je een event ergens vandaan haalt (bijv. uit de UI van de EventStore) dan moet het ondubbelzinnig zijn over welke aggregate(id) het gaat.

Reply:
Goed idee, nu is het een beetje omslachtig om het id uit de streamname te halen